Assets/HoloToolkit/XAML/Scripts/ApplicationViewManager.cs

```diff
@@ -0,0 +1,149 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using UnityEngine;
+#if WINDOWS_UWP
+using Windows.ApplicationModel.Core;
+using Windows.UI.Core;
+using Windows.UI.ViewManagement;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+#endif
+
+
+namespace HoloToolkit.Unity
+{
+    /// <summary>
+    /// ApplicationViewManager can switch to Plan View, populate an Application View (New Window of UAP), then navigate to the page with page name. 
+    /// After the page called 'CallbackReturnValue' method, the new Application View will be closed, and the system will switch back to your Full3D view.
+    /// The corotine were waiting will get the return value by callback.
+    /// </summary>
+
+    public class ApplicationViewManager : MonoBehaviour
+    {
+        void Start()
+        {
+#if WINDOWS_UWP
```

Comment:
I was just curious because I hadn't seen it is all. Thanks for the clarification guys.

btw, @waynebaby, I believe that this project only targets .net 3.5 currently. Will C# 6 work?